app/views/DirectoryView/Options.tsx

```diff
@@ -26,73 +26,75 @@ interface IDirectoryOptionsProps {
 	theme: TSupportedThemes;
 }
 
-export default class DirectoryOptions extends PureComponent<IDirectoryOptionsProps, any> {
-	private animatedValue: Animated.Value;
+const DirectoryOptions = memo(
```

Comment:
`memo` requires all props to be memoized in order to work properly.
By default on function components, functions like `toggleWorkspace` will be recreated on every render, which is going to make `memo` irrelevant here.
That said,
`DirectoryView` is a class component and `toggleWorkspace` uses arrow function, which makes it bound to the class and `memo` is going to end up working after all.

IMO we should remove `memo` from here, just because when we migrate `DirectoryView` to function component, we'll have to be remember to use `useCallback` on `toggleWorkspace` there in order to keep `memo` working here.
`DirectoryOptions` is not an expensive component, so I say we remove `memo`.